source/containers.tex

```diff
@@ -4079,10 +4079,12 @@
 \begin{note}
 Modifying any list requires access to the element preceding the first element
 of interest, but in a \tcode{forward_list} there is no constant-time way to access a
-preceding element. For this reason, ranges that are modified, such as those supplied to
-\tcode{erase} and \tcode{splice}, must be open at the beginning.
+preceding element.
+For this reason, \tcode{erase_after} and \tcode{splice_after}
+take fully-open ranges, not semi-open ranges.
 \end{note}
 
+%FIXME: Are these examples wrong?
```

Comment:
It looks like this FIXME is applying to the `forward_list` synopsis. Is the FIXME still necessary after the above change?
```suggestion
```

Reply:
Removed FIXME